### PR TITLE
Create aip-91.md

### PR DESCRIPTION
**Pull request by junkil-park**

### CODE REVIEW

File: aips/aip-91.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   * This addresses a long-standing and frequently requested feature in [this issue](https://github.com/aptos-labs/aptos-core/issues/6862).
+
+2. Eliminate Gas Fee Payment Requirement:
+   * Removes the requirement for a gas fee payer during simulations when the fee payer address is 0x0.
+   * Simulations will no longer validate the presence of sufficient funds for gas payment, allowing developers to test scenarios without considering gas fees.
+   * This addresses the feature request in [this issue](https://github.com/aptos-labs/aptos-core/issues/13686)
+
+3. Improve Multisig Simulation Consistency:
+   * Merges the Multisig payload simulation path with the execution path, ensuring consistency between simulation and actual execution.
+   * This resolves the issue of an onchain payload not being retrived (desribed [here](https://github.com/aptos-labs/aptos-core/issues/12703)).
```
Comment (banool) [inline aips/aip-91.md:35]: Nit: desribed -> described

Comment (junkil-park) [reply aips/aip-91.md:35]: Fixed. Thanks!

File: aips/aip-91.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+aip: 91
+title: Simulation Enhancement
```
Comment (davidiw) [inline aips/aip-91.md:3]:
```suggestion
title: Transaction Simulation Enhancement
```

File: aips/aip-91.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Standard (Ecosystem)
+created: 6/25/2024
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-91 - Simulation Enhancement
```
Comment (davidiw) [inline aips/aip-91.md:14]:
```suggestion
# AIP-91 - Transaction Simulation Enhancement
```

**Review by davidiw — APPROVED** [review]

**Review by thepomeranian — APPROVED** [review]

**Pull request merged**